code/modules/response_team/ert.dm

```diff
@@ -0,0 +1,369 @@
+// ERTs
+
+#define ERT_TYPE_AMBER		1
+#define ERT_TYPE_RED		2
+#define ERT_TYPE_GAMMA		3
+
+/datum/game_mode
+	var/list/datum/mind/ert = list()
+
+var/list/response_team_members = list()
+var/responseteam_age = 21 // Minimum account age to play as an ERT member
+var/datum/response_team/active_team = null
+var/send_emergency_team = 0
+var/ert_request_answered = 0
+
+/client/proc/response_team()
+	set name = "Dispatch CentComm Response Team"
+	set category = "Event"
+	set desc = "Send an CentComm response team to the station."
+
+	if(!check_rights(R_EVENT))
+		return
+
+	if(!ticker)
+		to_chat(usr, "<span class='warning'>The game hasn't started yet!</span>")
+		return
+
+	if(ticker.current_state == 1)
+		to_chat(usr, "<span class='warning'>The round hasn't started yet!</span>")
+		return
+
+	if(send_emergency_team)
+		to_chat(usr, "<span class='warning'>Central Command has already dispatched an emergency response team!</span>")
+		return
+
+	var/datum/nano_module/ert_manager/E = new()
+	E.ui_interact(usr)
+
+
+/mob/dead/observer/proc/JoinResponseTeam()
+	if(!send_emergency_team)
+		to_chat(src, "No emergency response team is currently being sent.")
+		return 0
+
+	if(jobban_isbanned(src, ROLE_ERT))
+		to_chat(src, "<span class='warning'>You are jobbanned from the emergency reponse team!</span>")
```

Comment:
feels like this should say `jobbanned from playing on an emergency response team`.

change or not, but you did typo `reponse` so at least fix that.